docs/source/model/dependencies.rst

```diff
@@ -711,27 +711,37 @@ to a CSV or JSON file containing the input payload.
 
 3. Update the model metadata
 ****************************
-Once you find the correct dependencies, you can create a new model with the correct dependencies.
-To do so, specify the ``extra_pip_requirements`` option when logging the model.
+Once you find the correct dependencies, you can update the logged model's dependency using :py:func:`mlflow.models.update_model_requirements` API.
 
 .. code:: python
 
     import mlflow
 
-    mlflow.pyfunc.log_model(
```

Comment:
question: Does `mlflow.pyfunc.log_model` with extra_pip_requirements still work for this purpose?